### PR TITLE
Refactor BEM style naming

### PR DESCRIPTION
**Pull request by ktkization**

### CODE REVIEW

File: webapp/web/configuration.css [hunk 1]
```diff
@@ -23,24 +23,28 @@
...
+.tag-list__tag-item {
     list-style-type: none;
     border-color: #ddd;
     border-bottom-style: solid;
     border-width: 1px;
     padding: 10px;
 }
 
-.configure-package__tag-item-active {
+.tag-list__tag-item_active {
```
Comment (marianamarasoiu) [inline webapp/web/configuration.css]: I think this should be:
```suggestion
.tag-list__tag-item--active {
```
(`active` seems to be a modifier on the tag item)

File: webapp/web/configuration.css [hunk 2]
```diff
@@ -132,19 +103,19 @@
...
     top: 20px;
 }
 
-.configure-package__tag-responses-item-drag-handle:hover {
+.tag-responses__item-drag-handle:hover {
     cursor: grab;
 }
 
-.configure-package__tag-responses-item-drag-handle-dragging {
+.tag-responses__item-drag-handle_dragging {
```
Comment (marianamarasoiu) [inline webapp/web/configuration.css]: should this be:
```suggestion
.tag-responses__item-drag-handle--dragging {
```
?

File: webapp/web/configuration.css [hunk 3]
```diff
@@ -23,24 +23,28 @@
...
+    background-color: #b5b3b3;
+}
+
+.tag-list__tag-item:hover {
     cursor: pointer;
     background-color: #b5b3b3;
 }
 
-.configure-package__button-add-tag-action {
+.tag-list__add-tag-action {
```
Comment (marianamarasoiu) [inline webapp/web/configuration.css]: this class and `.tag-responses__add-responses-action` look the same. It's likely we might want this button in other packages, so I suggest we can merge them into a single `.add-button` class. 

As a general suggestion - we don't need these to be explicitly part of the hierarchy, ie. we don't need to prefix them with `tag-list__` or `tag-responses__`. We can create more modular widgets/buttons/layouts that are named based on a more abstract role, not their content. That way we can also reuse them

File: webapp/web/configuration.css [hunk 4]
```diff
@@ -153,11 +124,44 @@
     padding: 2px;
 }
 
-.configure-package__tag-responses-item:focus {
+.tag-responses__item:focus {
     outline-color: #ddd;
 }
 
-.configure-package__button-add-responses-action {
+.tag-responses__add-tag-modal {
```
Comment (marianamarasoiu) [inline webapp/web/configuration.css]: I think we can remove the `tag_responses` prefix here and just use `add-tag-modal`, or even just `modal`? that way you can use the BEM strategy for the elements of the modal (`modal__heading`, modal__dropdown` etc.)

**Review by marianamarasoiu — APPROVED** [review]
thanks @ktkization - looks good, just a few suggestions

File: webapp/web/configuration.css (hunk 4 above)
Comment (ktkization) [reply webapp/web/configuration.css]: Changed it

File: webapp/web/configuration.css (hunk 3 above)
Comment (ktkization) [reply webapp/web/configuration.css]: I agree. Refactored it.

File: webapp/web/configuration.css (hunk 2 above)
Comment (ktkization) [reply webapp/web/configuration.css]: Yes, it's a modifier. Corrected.

File: webapp/web/configuration.css (hunk 1 above)
Comment (ktkization) [reply webapp/web/configuration.css]: Corrected.

**Review by marianamarasoiu — APPROVED** [review]
thanks @ktkization , merging this PR now :+1:

Can you then bring it into #8? thanks!

**Pull request merged**